### PR TITLE
RDCC-5868

### PR DESCRIPTION
**Pull request by vilasshelke-hmcts**
### Change description ###

eLinks : Create new mapping tables for Judicial Reference Data

https://tools.hmcts.net/jira/browse/RDCC-5867
https://tools.hmcts.net/jira/browse/RDCC-5868

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Comment by vilasshelke-hmcts** [comment]
Will reopen it

**Review by AdilOoze — APPROVED** [review]

**Review by nayeemshaik-hmcts — COMMENTED** [review]
dbjudicialdata.jrd_lrd_region_mapping file contains pasted thrice. Could you please correct it

**Comment by vilasshelke-hmcts** [comment]
> 

Accommodated the changes. Thanks @nayeemshaik-hmcts

**Review by kavithagade12 — APPROVED** [review]

File: src/main/resources/db/migration/V1_41__create_elinks_schema.sql [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+CREATE TABLE dbjudicialdata.region_type (
+	region_id varchar(64) NOT NULL,
+	region_desc_en varchar(256) NOT NULL,
+	region_desc_cy varchar(256) NULL,
+	CONSTRAINT region_id PRIMARY KEY (region_id)
+);
+
+-- Create table script : judicial_role_type
+CREATE TABLE dbjudicialdata.judicial_role_type (
+	role_id integer NOT NULL,
```
[the remark this answers is not in the document]
Comment (vilasshelke-hmcts) [reply src/main/resources/db/migration/V1_41__create_elinks_schema.sql]: > 

@kavithagade12 - This is another story which @shivajihmcts is looking into it. I am deleting it from my commit.

**Review by AdilOoze — APPROVED** [review]

**Review by nayeemshaik-hmcts — APPROVED** [review]

**Review by nayeemshaik-hmcts — APPROVED** [review]

**Pull request merged**